### DISCUSSION TITLE
Can't use A1111 on Colab due to warning?

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#2428] [General]
Hi. I have returned to SD after a few months and I am using TheLastBen colabs, but it is giving me the same "Not allowed" warning. Is this the end for running A1111 GUI on Free Colab? I tried encoding all possible keywords that could trigger this warning but still same warning. The same trick worked few months back. Please help.

((( I am broke, cant afford to use runpod or some other server to use A1111. Please give a solution to run it on colab.)))) 

The code:

```
#@markdown # Install/Update Repo
from IPython.utils import capture
from IPython.display import clear_output
from subprocess import getoutput
import ipywidgets as widgets
import sys
import fileinput
import os
import time
import base64
import gdown
from gdown.download import get_url_from_gdrive_confirmation
import requests
from urllib.request import urlopen, Request
from urllib.parse import urlparse, parse_qs, unquote
from tqdm import tqdm
import six


gg=base64.b64decode(("c3RhYmxlLWRpZmZ1c2lvbi13ZWJ1aQ==").encode('ascii')).decode('ascii')
apex = base64.b64encode(("c2Q=").encode('ascii')).decode('ascii')
gu = base64.b64encode(("QVVUT01BVElDMTE=").encode('ascii')).decode('ascii')
pt = base64.b64encode(("L2NvbnRlbnQvZ3JpdmU=").encode('ascii')).decode('ascii')
ppp = base64.b64encode(("TXlEcmVpdmU=").encode('ascii')).decode('ascii')  
li = base64.b64encode(("aHR0cHM6Ly9naXRodWIuY29t").encode('ascii')).decode('ascii')


if not os.path.exists("$pt"):
  print('[1;31mGdrive not connected, using colab storage ...')
  time.sleep(4)
  pth=$ppp
  !mkdir -p $pt/$pth
  Shared_Drive=""

if Shared_Drive!="" and not os.path.exists("$pt/Shareddrives"):
  print('[1;31mShared drive not detected, using default My Drive')
  pth=$ppp

with capture.capture_output() as cap:
  def inf(msg, style, wdth): inf = widgets.Button(description=msg, disabled=True, button_style=style, layout=widgets.Layout(min_width=wdth));display(inf)
  fgitclone = "git clone --depth 1"
  %mkdir -p $pt/$pth/$apex
  %cd $pt/$pth/$apex
  !git clone -q --branch master $li/$gu/$gg
  !mkdir -p $pt/$pth/$apex/$gg/cache/
  os.environ['TRANSFORMERS_CACHE']=f"$pt/{pth}/$apex/$gg/cache"
  os.environ['TORCH_HOME'] = f"$pt/{pth}/$apex/$gg/cache"

with capture.capture_output() as cap:
  %cd $pt/$pth/$apex/$gg
  !git reset --hard
  !git checkout master
  time.sleep(1)
  !rm webui.sh
  !git pull
clear_output()
inf('\u2714 Done','success', '50px')

#@markdown ---
```

**Comment by ClaraSanders** [comment]
Google banned the free use of SD like one or two months ago, sorry. There is no free solution.

**Reply by FIRELORD1988** [reply]
i was using A1111 in my pc, My PC is acting harshly.", i decided to subscribe in pro + Colab , but Currently only available to subscribers with a US billing address.
any advice ?
note: my google drive storage is 2TB if this help

**Reply by TheLastBen** [reply]
Use the alternatives, Runpod or Paperspace

**Comment by abc123desygn** [comment]
is there any way to use SD with the free version of Google Colab?

**Reply by i-huzaifa-arshad** [reply]
No, unfortunately. But I will find one. Stay connected with me.

**Reply by TheLastBen** [reply]
You can, just use the latest notebook, you can even run SDXL on free colab

**Reply by abc123desygn** [reply]
When was support for the free colab added back?

**Reply by TheLastBen** [reply]
a few days ago, let's hope google guys won't find out soon

**Reply by abc123desygn** [reply]
Cool! Thanks!

**Reply by i-huzaifa-arshad** [reply]
Thats a great news.

**Comment by ClaraSanders** [comment]
> You can, just use the latest notebook, you can even run SDXL on free colab

Am I dreaming? Really? I tried it even using my computing units and A100 GPU and never worked.

**Reply by TheLastBen** [reply]
Try again with a fresh SD folder and a T4

**Reply by ClaraSanders** [reply]
It works! It's amazing, really. I thought I'd have to wait until some kind of polished version was released. Now, the issue I'm facing is that pictures look a bit odd. I guess I need a VAE and should adjust some settings. Could you tell me if I need to make changes in the configuration? Which samplers are compatible with the XL? How about the clip?

**Reply by TheLastBen** [reply]
you don't need a different vae or a different text encoder, all you need is Loras for your specific purposes, try a few from here https://civitai.com/user/TheLastBen/models

**Reply by ClaraSanders** [reply]
Oh, I see. So it is normal for XL to look sometimes weird? I don't know how to describe it: plastic-like skin, SD 1-4 hands, extremely blurry backgrounds...I really like the composition and the variety of poses/scenarios but that is all, the rest is appalling.

**Reply by TheLastBen** [reply]
yes, that's SDXL, but it can easily be improved with loras
